articles/azure-app-configuration/enable-dynamic-configuration-dotnet-core-push-refresh.md

```diff
@@ -41,7 +41,9 @@ In this tutorial, you learn how to:
 
 ## Prerequisites
 
-To do this tutorial, install the [.NET Core SDK](https://dotnet.microsoft.com/download).
+* Install [.NET Core SDK](https://dotnet.microsoft.com/download).
```

Comment:
Remove. If people did the quickstart, they have this already.

Reply:
removed